### ISSUE TITLE
[Bug]: TypeError: Cannot use 'in' operator to search for 'CallExpression' in undefined (beta 55)

### ISSUE THREAD

**Issue by elliotgonzalez-lk** [storybookjs/storybook#21280]
### Describe the bug

Fresh install of NextJs.  Ran `npx sb@next upgrade --prerelease` and after running `yarn storybook` now getting:

`TypeError: Cannot use 'in' operator to search for 'CallExpression' in undefined `

Upgraded from beta 48 to beta 55.  Beta 48 worked without issue.

### System

```shell
Environment Info:
  System:
    OS: macOS 13.2.1
    CPU: (10) arm64 Apple M1 Max
  Binaries:
    Node: 18.12.1 - /usr/local/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 9.2.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 110.0.5481.100
    Firefox: 110.0
    Safari: 16.3
  npmPackages:
    @storybook/addon-essentials: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/addon-interactions: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/addon-links: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/addon-mdx-gfm: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/blocks: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/nextjs: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/react: ^7.0.0-beta.55 => 7.0.0-beta.55
    @storybook/testing-library: ^0.0.14-next.1 => 0.0.14-next.1
```

**Comment by elliotgonzalez-lk** [comment]
Fixed this by deleting node-_modules and yarn.lock

**Comment by montchr** [comment]
I’m also running into this on the latest beta. I’d guess this may come up again for others too.

**Comment by marchdoe** [comment]
Running `pnpm` updated to `7.0.0-beta.55` and also resulted in this error. Was not fixed by removing node_modules and lockfiles.

**Comment by yannbf** [comment]
Can any of you provide a reproduction? That would help immensely!

**Comment by shilman** [comment]
@yannbf we have a repro from @JReinhold in SvelteUI

**Comment by tortilaman** [comment]
I upgraded to `7.0.0-beta.55` and encountered this issue. When I downgraded to `7.0.0-beta.54` the issue was resolved. It looks like this is limited to the latest release.

**Comment by MetheusX** [comment]
Can confirm removing `node_modules` and `yarn.lock` and reinstalling modules fixed this for me after upgrading from `7.0.0-beta.46` to `7.0.0-beta.56`. Thank you for help

**Comment by shilman** [comment]
Boo-yah!! I just released https://github.com/storybookjs/storybook/releases/tag/v7.0.0-beta.57 containing PR #21301 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb@next upgrade --prerelease
```

**Comment by shilman** [comment]
Whoopee!! I just released https://github.com/storybookjs/storybook/releases/tag/v7.0.0-beta.58 containing PR #21326 that references this issue. Upgrade today to the `@next` NPM tag to try it out!

```
npx sb@next upgrade --prerelease
```

**Comment by FloodGames** [comment]
> Can confirm removing `node_modules` and `yarn.lock` and reinstalling modules fixed this for me after upgrading from `7.0.0-beta.46` to `7.0.0-beta.56`. Thank you for help

Somehow needed now with the latest version of babel

**Comment by jaredly** [comment]
fwiw for me, just removing `node_modules` (didn't need to remove `yarn.lock`) and re-running `yarn` got rid of this strange error 🤷
